src/main/java/fr/polytech/sim/log/NamedLogger.java

```diff
@@ -16,4 +16,12 @@ protected NamedLogger(String name) {
         this.name = name;
     }
 
+    public void log(String format, Object... args) {
+        String entry = String.format(format, args);
+        String message = String.format("%s\t%s\n", this.name, entry);
+        this.output(message);
+    }
+
+    public abstract void output(String message);
```

Comment:
C'est préférable de garder la méthode `protected` pour éviter une utilisation non agréée.
```suggestion
    protected abstract void output(String message);
```